gui/index.html

```diff
@@ -911,6 +918,25 @@ <h4 translate class="modal-title">Settings</h4>
     </div>
   </div>
 
+  <!-- Panic reporting modal -->
+
+  <div id="pr" class="modal fade" data-backdrop="static" data-keyboard="false" tabindex="-1">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="modal-header alert alert-success">
+          <h4 translate class="modal-title">Automatically send anonymised fatal error logs to developers?</h4>
```

Comment:
Title case, probably "Allow Anonymous Error Reporting?" to conform to the existing wording on usage reporting.